src/pages/index.js

```diff
@@ -78,14 +134,26 @@ const profileEditPopup = new PopupWithForm(
   {
     popupSelector: "#profile-edit-modal",
     handleFormSubmit: ({ title, description }) => {
-      profileInfo.setUserInfo(title, description);
-      profileEditPopup.close();
-    }
+      profileEditPopup.renderLoading(true);
+      api.updateProfile(title, description)
+        .then((data) => {
+          profileInfo.setUserInfo(data.name, data.about,data.avatar),
+            profileEditPopup.close();
```

Comment:
* You need to format the markup (and code) to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`

`,` is not needed at the end of line 140: it brakes the formatting

```js
      .then((data) => {
        profileInfo.setUserInfo(data.name, data.about, data.avatar);
        profileEditPopup.close();
      })
      .catch((err) => {
```